LOGGING.md

```
@@ -0,0 +1,687 @@
+|-------|-----------------|----------|
+| `debug` | Everything including LLM prompt summaries | Local development only |
+| `info` | Standard operational logs | Default — all environments |
+| `warn` | Warnings and errors only | - |
+| `error` | Errors only | - |
+
+> **Never set `LOG_LEVEL=debug` in production.** Debug output may include LLM prompt
+> summaries and other sensitive operational data.
+
+**Development — `NODE_ENV=development`** activates `pino-pretty`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)